assert/opt/opt_test.go

```diff
@@ -167,6 +168,7 @@ func (p *pathRecorder) record(path gocmp.Path) bool {
 func matchPaths(fixture interface{}, filter func(gocmp.Path) bool) []string {
 	rec := &pathRecorder{filter: filter}
 	gocmp.Equal(fixture, fixture, gocmp.FilterPath(rec.record, gocmp.Ignore()))
+	sort.Strings(matches)
```

Comment:
Looks like go-cmp filter now may result in results to be returned in
random random order; this sorts the results before comparing, but I'm
not sure if order is important (in which case this could actually be
a bug)